### DISCUSSION TITLE
Is it possible to run 'dagger run python example.py' in parallel?

### DISCUSSION THREAD

**Discussion by carles-aiven** [dagger/dagger#6480] [Help]
Hello,
I'm considering using dagger inside a Jenkinsfile. We use the Jenkinsfile to define different stages, some of them running in parallel. To integrate dagger, we could call 'dagger run python tests.py', 'dagger run python checks.py', etc in different Jenkins parallel stages.

Does the dagger engine support this?

Thanks in advance

**Comment by jedevc** [comment]
Yup, you should definitely be able to do this :tada:

If you are targeting each stage at the same dagger engine, then the builds will all get done in parallel - additionally, any common items of work between each of the stages, should get deduplicated, so nothing should get done twice :tada:
